addon-test-support/test.js

```diff
@@ -80,7 +80,9 @@ export default function test(testName, callback) {
           testTimeoutDeferred.resolve();
           resolve(data);
         } else {
-          setTimeout(poll, 10, resolve, reject);
+          setTimeout(function() {
```

Comment:
I certainly can do that. But I am not sure I understand the need for that. `poll` does not use `this`, so the scope is not relevant, is it? And before and after the change `this` would be the global `window` object, so nothing has changed. Or do I miss something?